bodhi/tests/server/services/test_updates.py

```diff
@@ -509,17 +510,18 @@ def test_provenpackager_request_privs(self, publish, *args):
             topic='update.request.testing', msg=mock.ANY)
 
         build = self.db.query(RpmBuild).filter_by(nvr=nvr).one()
+        build.ci_status = CiStatus.passed
         eq_(build.update.request, UpdateRequest.testing)
 
         # Try and submit the update to stable as a non-provenpackager
         app = TestApp(main({}, testing=u'ralph', session=self.db, **self.app_settings))
         post_data = dict(update=nvr, request='stable',
                          csrf_token=app.get('/csrf').json_body['csrf_token'])
         res = app.post_json('/updates/%s/request' % str(nvr), post_data, status=400)
-
```

Comment:
This change seems unrelated to the goal of the pull request - I prefer to keep commits atomic.